src/lib.rs

```diff
@@ -39,6 +39,16 @@ impl<S> Snapshot<S>
 where
     S: Snapable + Debug + DeserializeOwned + PartialEq + Serialize,
 {
+
+    pub fn new(file: String, module_path: String, test_function: String, recorded_value: S) -> Self {
+        Snapshot {
+            file: file.replace("\\", "/"),
```

Comment:
What do you think about storing the [components](https://doc.rust-lang.org/std/path/struct.Path.html#method.components) of the project-relative path in this struct? It doesn't bless any particular filesystem representation and handles weird cases I can't think of right now.

Reply:
Yeah that works for me.  I can update the PR.  👍